recipes/atactk/build.sh

```diff
@@ -0,0 +1,2 @@
+#!/bin/bash
+pip install .
```

Comment:
Use that here: https://github.com/bioconda/bioconda-recipes/blob/eaf59a144191c3989ee0c2bd0d799fb0a9281f3e/recipes/bracken/build.sh#L5 for your scripts.